### PR TITLE
routes set, new pages made

### PR DESCRIPTION
**Pull request by bjameson72**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import "./App.css";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
+import GameThread from "./pages/GameThread";
```
Comment (ScotteRoberts) [inline client/src/App.js:4]: You might want to put all of these page imports into a single file and import that as an object so you don't have a ton of "import page 1, import page 2, ..."

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <div>
```
Comment (ScotteRoberts) [inline client/src/components/Footer.js:6]: Div soup!

File: client/src/pages/GameThread.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Header from "../components/Header";
+import "../styles/GameThread.css";
+
+class GameThread extends React.Component {
+  render() {
+    return (
+      <div className="weirdContainer">
+        {/* <Header /> */}
+        <div className="Title">PUT GAMETHREAD HERE</div>
```
Comment (ScotteRoberts) [inline client/src/pages/GameThread.js]: This should be a lowercase "title" className to be consistent

File: client/src/pages/Leaderboard.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Header from "../components/Header";
+
+class Leaderboard extends React.Component {
+  render() {
+    return (
+      <div>
+        <div className="Title">Wassup dudes</div>
```
Comment (ScotteRoberts) [inline client/src/pages/Leaderboard.js]: This should be a lowercase "title" className to be consistent

File: client/src/pages/Login.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Header from "../components/Header";
+
+class Login extends React.Component {
+  render() {
+    return (
+      <div>
+        {/* <Header /> */}
+        <div className="Title">The routes are set</div>
```
Comment (ScotteRoberts) [inline client/src/pages/Login.js]: This should be a lowercase "title" className to be consistent

File: client/src/styles/GameThread.css [hunk 6]
```diff
@@ -0,0 +1,18 @@
+.Title {
```
Comment (ScotteRoberts) [inline client/src/styles/GameThread.css]: This should be a lowercase "title" class to be consistent

File: client/src/styles/Header.css [hunk 7]
```diff
@@ -32,7 +32,7 @@ nav {
 
 .linkNames {
   margin-left: 0rem;
-  color: #f9f8f7;
+  color: #a8a8a8;
```
Comment (ScotteRoberts) [inline client/src/styles/Header.css:35]: I would suggest starting a color palette if you do not already have one.

File: client/src/components/Footer.js (hunk 2 above)
Comment (bjameson72) [reply client/src/components/Footer.js:6]: HAHAHA on purpose :p

File: client/src/pages/GameThread.js (hunk 3 above)
Comment (bjameson72) [reply client/src/pages/GameThread.js]: thanks!

File: client/src/pages/Leaderboard.js (hunk 4 above)
Comment (bjameson72) [reply client/src/pages/Leaderboard.js]: got eem

File: client/src/pages/Login.js (hunk 5 above)
Comment (bjameson72) [reply client/src/pages/Login.js]: got it!

File: client/src/styles/GameThread.css (hunk 6 above)
Comment (bjameson72) [reply client/src/styles/GameThread.css]: yup!

File: client/src/styles/Header.css (hunk 7 above)
Comment (bjameson72) [reply client/src/styles/Header.css:35]: should be coming soon! -- in the next day or so, just wanted to get this up

File: client/src/App.js (hunk 1 above)
Comment (bjameson72) [reply client/src/App.js:4]: agreed!

**Comment by jonazali** [comment]
Reviewed and Web Application Launched Successfully, Had Scott's tips integrated.

**Review by jonazali — APPROVED** [review]
Reviewed and Launch Successfully.

**Pull request merged**